fredshema/hoop-mobile-app
# feat: add notification feature

**What does this PR do?**
Display notification info.

**Description of Task to be completed?**
Use appwrite Database sdk to display notifications from the backend.

**How should this be manually tested?**
Navigate to the home page and click on the notification bell icon. You will be redirected to the notifications screen and a loading screen will appear. Wait a bit and a list of notifications will be displayed. To refresh the list try pulling the page from the top to simulate a refresh action.
 
**What are the relevant pivotal tracker/Trello stories?**
https://trello.com/c/ypVRxjaX/41

**Questions:**
N/A
